src/videojs.ads.js

```diff
@@ -403,7 +409,6 @@ var
               },
               'adtimeout': function() {
                 this.state = 'content-playback';
-                player.play();
```

Comment:
This was no longer necessary.  This was caught in the 'starts the content video if there is no preroll' test which checks the number of times play is called.